## This message contains the information for an object in a physical map
Header header
string name

geometry_msgs/Pose pose
geometry_msgs/Pose previous_pose

### very specific for how people are currently conceptualized as an ellipse in a 2d space.
float64 ellipse_a
float64 ellipse_b
float64 ellipse_theta